specs/fulu/beacon-chain.md

```diff
@@ -20,6 +25,89 @@ and is under active development.
 
 ## Configuration
 
+### Blob schedule
+
+*[New in EIP7892]* This schedule defines the maximum blobs per block limit for a
+given epoch.
+
+*Note*: The blob schedule is to be determined.
+
+<!-- list-of-records:blob_schedule -->
+
+| Epoch | Max Blobs Per Block | Description |
+| ----- | ------------------- | ----------- |
+
+## Helper functions
+
+### Misc
+
+#### `compute_fork_digest`
+
+*Note:* The `compute_fork_digest` helper is updated to account for
+Blob-Parameter-Only forks.
+
+```python
+@dataclass
+class BlobScheduleEntry:
+    epoch: Epoch
+    max_blobs_per_block: uint64
```

Comment:
This will need to exist in its own section somehow, named "New `BlobScheduleEntry`". Despite what I said on the EIP, I think it should probably be a Container. Sort of like this: https://github.com/ethereum/consensus-specs/blob/74a077e83c08bddf369f8d5f3b3955cd57f201d8/specs/electra/beacon-chain.md?plain=1#L218-L222